frontend_tests/puppeteer_tests/settings.ts

```diff
@@ -355,7 +355,6 @@ async function test_default_language_setting(page: Page): Promise<void> {
     await page.goto("http://zulip.zulipdev.com:9981/#settings"); // get back to normal language.
     await page.waitForSelector(display_settings_section, {visible: true});
     await page.click(display_settings_section);
-    await page.waitForSelector("#language-settings-status", {visible: true});
```

Comment:
We already check that save alert indicator appears on line:354. So no need to assert its visibility again as it is hidden by default(when it has no content).

Reply:
Makes sense.  As a side note, in English parentheticals should always have a space before them, so:

> by default(when it has no content).

should be

> by default (when it has no content).

It's not particularly important, but I have been finding it distracting whenever I read it, so I think it is worth tweaking your writing style :)